data/human/Republic Paradise Threatened.TXT

```diff
@@ -0,0 +1,301 @@
+			`	Before you can respond the light starts moving, faster and faster until it is spelling out a message.`
+			`	"1,000,000 credits to seal your hatch and power down your sensors"`
+			choice
+				`	Do as they wish. A million credits is a million credits.`
+				`	Darken the viewports and ignore the message.`
+					end
+				`	You press the button to seal the outer hatch and then, after a moments hesitation, power the sensors down completely. As you sit in your chair you wonder what is going on outside but the chime of your comm which tells you that one million credits has been added to your account dampens down your curiosity significantly.`
+				`	After what seems like an age but was apparently only five minutes, a green laser spot appears on your console and just as quickly disappears again. You take that as an 'all clear' and look out your viewport. The ground crew are still working but there is no sign of Agent Norris. You open the hatch and take a look around but the agent is nowhere to be seen.`
+				`	Without knowing what other systems you are supposed to be investigating you can't continue the mission. You tell yourself that the truth about the Wolf Pack nukes has to come out eventually, and in the mean time at least you have a million more reasons to sleep well.`
+					payment 1000000
+					log "Accepted a bribe to look the other way instead of tracing the source of the Wolf Pack nukes. The money was just too good to turn down."
+					fail
+	on enter
+		conversation
+			`Agent Norris instructs you to plot a course for <destination> and seems completely unaware of the odd message that you received.
+			choice
+				`	Tell him what happened.`
+				`	No need to worry him.`
+					goto onward
+				`	You decide that he should know and explain that somebody used a laser to send you a message asking you to seal the hatch and deactivate the sensors. Agent Norris nods slowly.
+				`	"What did they offer you?" he asks as though he is aksing what you had for your lunch.
+				`	"A million credits," you say.`
+				`	He nods again looking mildly impressed. "And you turned it down? Captain <last>, I didn't know you cared."`
+				`	"Wait, was that a joke?" you ask, but Norris' face is still stony.`
+				`	"Thank you, Captain," he says. "If you had accepted I have no doubt I would be dead by now. That or under interrogation. We must proceed more carefully from now on. If we can avoid refueling on any Syndicate worlds we should do so."`
+				label onward
+					`	You program the nav computer and prepare to jump, but before you can do so Agent Norris receives a message and raises a hand as though asking for you to wait. He reads the message several times before looking back up at you.`
+					`	"The injunction has been approved," he says. "The Syndicate won't be so friendly from here on out."`
+					`	You laugh. If that was the Syndicate being 'friendly' you wouldn't want to see them hostile, but it looks like you're going to.`

```

Comment:
I wouldn't laugh in this situation, the Syndicate is directly willing to kill an agent of Republic Intelligence, which seems more serious than just some old merchant getting plundered by a pirate. 

You shouldn't tell the player what they are feeling.